src/components/form-errors.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function FormError (props) {
```

Comment:
I offer you the following refactoring:
1. we should extract `DuplicateError` component because I see the `render` method now and understand that depends on `fieldName` the result has a few in common
And also `duplicateUser` is not a real field it's just a type of error.
1. we'd better to implement a component such as e.g. `ErrorWrap` or `InputWithError`. So we should use it to show the error only. This component should get only one property `error` (now we pass to much - all `formErrors` and `fieldName`) and its children (input+label or even a whole `p` in our case) in `children` property https://ru.reactjs.org/docs/glossary.html#propschildren